tests/integration/test_error.py

```diff
@@ -4,28 +4,12 @@
 """Integration tests for WordPress charm in error."""
 
 import pytest
-import pytest_operator.plugin
 
-from .constants import BLOCKED_STATUS_NAME
+from tests.integration.helper import WordpressApp
 
 
-@pytest.mark.usefixtures("build_and_deploy")
-@pytest.mark.asyncio
-@pytest.mark.usefixtures("app_config")
-@pytest.mark.parametrize(
-    "app_config",
-    [
-        {
-            "db_host": "test_db_host",
-            "db_name": "test_db_name",
-            "db_user": "test_db_user",
-            "db_password": "test_db_password",
-        }
-    ],
-    indirect=True,
-    scope="function",
-)
-async def test_incorrect_db_config(ops_test: pytest_operator.plugin.OpsTest, application_name):
+@pytest.mark.usefixtures("wordpress")
```

Comment:
Is the `usefixtures` required given that it is being taken as an argument?

Reply:
Yes, it's unnecessary, removed, thanks!